src/Samples/VisualBasic/RemoveByVal/Impl/project.json

```diff
@@ -1,8 +1,6 @@
 ﻿{
   "dependencies": {
-    "Microsoft.Composition": "1.0.27",
-    "System.Collections.Immutable": "1.1.37",
-    "System.Reflection.Metadata": "1.2.0"
+    "Microsoft.Composition": "1.0.27"
```

Comment:
You might be able to already remove these. If the project is referencing Workspaces, then this would already flow.